.github/workflows/release-workflow.yml

```diff
@@ -1,41 +1,33 @@
+
 name: Release
 on:
   push:
+    branches: ['master', 'main']
     tags:
-      - 'release-*'
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
Note that the axion-release-plugin changed the default `prefix` setting for version creation from `release-` to `v`. Since we only have one prior version in this repository, I chose to simply switch than update to use the "legacy" tag.